doc/pz-compute.md

```diff
@@ -12,130 +12,85 @@ ssh loginapl01
 
 ### Quick setup 
 
+0. Copy the file 
+
+   ```shell
+    cp <path-to-y=pz-compute>/pz-compute/scheduler_examples/slurm/setup/prod-config ~/.prod-config"
+    ```
+
 1. Add the following block to your `~/.bashrc` file:
     
     ```shell
-    if [[ -d ~app.photoz ]]
-    then
-        source ~app.photoz/conf-pz-compute-user.sh
-    fi
+    alias pz-prod=". ~/.prod-config"
     ```
 
 2. Logout and login again to **loginapl01**. 
 
+3. Type pz-prod, this is going to send you to your t0 scratch area. Once you are here, everytime you login, just need to run this command.
+
+    ```shell
+    pz-prod
```

Comment:
That error happens because yaml lib is not installed into the environment. Once this issue is closed we should install the lib into the default production env